### PR TITLE
feature/fbfw-69/game-phase-icon

### PR DESCRIPTION
**Pull request by jchoi166**
Demo link: 
https://fbfw-69-gamephaseicons.netlify.app/

### CODE REVIEW

**Review by hyukcodazen — APPROVED** [review]
LGTM!
Maybe we can think about "enabled" prop to be handled within component as a state.

**Review by nebevets — CHANGES_REQUESTED** [review]
just a few nits to clean up

**Comment by jchoi166** [comment]
@bryancodazen LOL I though it was weird that Hyuk reviewed my PR instead of you. I guess I mis-clicked on the name. 

Everything has been updated with the requested changes. There are a lot of commits on this PR because I initially requested to merge this branch into FBFW-60 like a dum dum for some reason. I changed it so that its merging back into staging, but I had to clean up the code a bit so that it wouldn't have any merge conflicts.

File: beta-src/src/enums/Season.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+enum Season {
+  AUTUMN = "Autumn",
+  SPRING = "Spring",
+  WINTER = "Winter",
+  ENABLED = "Enabled",
```
Comment (bryancodazen) [inline beta-src/src/enums/Season.ts:5]: This doesn't belong here.

File: beta-src/src/components/svgr-components/WDGamePhaseIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import * as React from "react";
+import { Box, Button } from "@mui/material";
+import Season from "../../enums/Season";
+
+interface GamePhaseIconProps {
+  disabled?: boolean;
+  iconState: Season;
```
Comment (bryancodazen) [inline beta-src/src/components/svgr-components/WDGamePhaseIcon.tsx:7]: I don't know that I understand the relationship between iconState and seasons. Naming seems off here.

**Review by bryancodazen — CHANGES_REQUESTED** [review]

**Comment by bryancodazen** [comment]
> @bryancodazen LOL I though it was weird that Hyuk reviewed my PR instead of you. I guess I mis-clicked on the name.
> 
> Everything has been updated with the requested changes. There are a lot of commits on this PR because I initially requested to merge this branch into FBFW-60 like a dum dum for some reason. I changed it so that its merging back into staging, but I had to clean up the code a bit so that it wouldn't have any merge conflicts.

It's great that others review the PRs, when creating a PR you want to add everyone working on web dip as reviewers.

**Comment by jchoi166** [comment]
@nebevets Github isn't letting me re-request a review from you, so pinging you here!

**Review by bryancodazen — APPROVED** [review]

**Review by nebevets — APPROVED** [review]
LGTM

**Pull request merged**